Assets/KoboldKare/Scripts/Equipment/EquipmentSkinnedMesh.cs

```diff
@@ -1,81 +1,62 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using JigglePhysics;
 using Naelstrof.Inflatable;
+using PenetrationTech;
 using UnityEngine;
 
 [CreateAssetMenu(fileName = "New SkinnedMeshEquipment", menuName = "Equipment/SkinnedMesh Equipment", order = 1)]
 public class EquipmentSkinnedMesh : Equipment {
-    [SerializeField] private GameObject prefabContainingSkinnedMeshRenderers;
-    private List<SkinnedMeshRenderer> targets;
-
-    private class BlendShapeCopier : MonoBehaviour {
-        public SkinnedMeshRenderer source;
-        private List<SkinnedMeshRenderer> targets;
-
-        void Awake() {
```

Comment:
What happened here? We've deleted all the work from the previous commits? Is that a mistake?

Reply:
was a goof on my part because of me fumbling over github desktop. will send a new proper PR later